include/srtp.h

```diff
@@ -1484,6 +1697,18 @@ srtp_err_status_t srtp_set_debug_module(char *mod_name, int v);
  */
 srtp_err_status_t srtp_list_debug_modules(void);
 
+/**
+ * @brief srtp_get_protect_data_added_length(policy, use_mki, mki_index, is_rtcp, length)
+ *
+ * Determines the length of the amount of data Lib SRTP will add to the 
+ * packet during the protect process. The length is returned in the length parameter
+ *
+ * returns err_status_ok on success, err_status_bad_mki if the MKI index is invalid
+ *
+ */
+srtp_err_status_t srtp_get_protect_data_added_length(const srtp_policy_t *policy, uint32_t use_mki,
```

Comment:
Sounds fair.
I would still prefer SRTP_MAX_TRAILER_LEN  to be defined as SRTP_MAX_TAG_LEN + SRTP_MAX_MKI_LEN since it was previously documented to do so.

Also having two functions that operate on session to match the rest of the API.
srtp_get_protect_trailer_length()
srtp_get_protect_rtcp_trailer_length()